packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -89,6 +89,7 @@ export default class QuanticSearchBox extends LightningElement {
    * @param {SearchEngine} engine
    */
   initialize = (engine) => {
+    this.engine = engine;
```

Comment:
Out of curiosity, why do you need to set `this.engine`? Could you not dispatch the action directly from `engine`?

Reply:
this is the current pattern used everywhere else in quantic in order to dispatch as follows:

<img width="301" alt="image" src="https://github.com/user-attachments/assets/703138c2-e36b-4227-9328-5c2eda7d300f">